.gitignore

```diff
@@ -34,3 +34,5 @@ build
 
 # STAC schema cache
 stac_schema_cache.sqlite
+
+_version.py
```

Comment:
There's two mentions of `_version.py` in `.gitattributes`, I'm guessing those should be removed if the file is being ignored.

Reply:
Yep, tidied up.